docs/specs/epic1/story12.md

```
@@ -1,143 +1,257 @@
-- AC1: Un fichier `backend/Dockerfile` est créé.
-- AC2: Le Dockerfile utilise une approche multi-stage :
-    - Un stage "builder" basé sur une image Maven (ex: `eclipse-temurin:21-jdk-jammy` ou `maven:3.9-eclipse-temurin-21`) qui copie les fichiers sources du backend, et exécute `mvn clean package -DskipTests` pour générer le JAR.
-    - Un stage "runner" basé sur une image JRE appropriée (ex: `eclipse-temurin:21-jre-jammy`).
-- AC3: Le stage "runner" copie le fichier JAR exécutable du stage "builder".
-- AC4: Un utilisateur non-root est créé et utilisé pour exécuter l'application Spring Boot dans le stage "runner".
-- AC5: Le port de l'application Spring Boot (ex: 8080) est exposé par le conteneur.
-- AC6: L'image Docker peut être buildée avec succès (`docker build -t spring-backend-test ./backend`).
-- AC7: Un conteneur lancé à partir de l'image buildée démarre l'application Spring Boot avec succès (les logs doivent l'indiquer). L'endpoint `/actuator/health` doit être accessible (si le conteneur est lancé avec le port mappé).
+-   AC1: Un fichier `backend/Dockerfile` est créé.
+-   AC2: Le Dockerfile utilise une approche multi-stage optimisée :
+    -   Un stage "builder" basé sur `maven:3.9.8-eclipse-temurin-21@sha256:6847cbbf21e159f97819f18336cf6bbc24a89f9eb6439317520d93f904e9a916` qui exécute `mvn -B clean package -Dmaven.test.skip=true -Dmaven.wagon.http.retryHandler.count=3` pour générer le JAR.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)